### ISSUE TITLE
Stremio Web not working or connecting to server 

### ISSUE THREAD

**Issue by itsmhashim** [Stremio/stremio-bugs#949]
**Describe the bug**
Stremio web is not working at all and not connecting to server. I have tried installing stremio service as well but it still doesn't work. Whenever I play any stream it says "stream not supported" and the server status is always showing "error" in settings. I have tried different browsers, different devices, uninstalling and reinstalling stremio service, clearing cache and cookies of the browser but no use. There's no firewall or vpn enabled. 

**Screenshots**
![IMG_20241012_153321](https://github.com/user-attachments/assets/7e4220f1-430c-4804-b3ff-9fb686245242)
![IMG_20241012_153356](https://github.com/user-attachments/assets/04baa154-940c-454f-91b7-03d9a50065e9)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Brave
 - Versio: latest

**Comment by TRtomasz** [comment]
what do you see when you open http://127.0.0.1:11470/ if your browser?

**Comment by steviator** [comment]
Experienced the same problem, Brave is blocking something needed for this, click on Lionface in the toolbar and turn off shields for app.stremio.com.

**Comment by daaf85** [comment]
i have a simular problem but I can watch streaming content. the issue I have experience is an error in the webbrowser version in safari on my macOS. In the settings I see an 'error' by streaming status. 

when I open Stremio in de webbrowser in google chrome I don't have this problem. is this normal?

**Comment by itsmhashim** [comment]
> what do you see when you open http://127.0.0.1:11470/ if your browser?

I don't see anything the page doesn't open

**Comment by itsmhashim** [comment]
> i have a simular problem but I can watch streaming content. the issue I have experience is an error in the webbrowser version in safari on my macOS. In the settings I see an 'error' by streaming status.
> 
> when I open Stremio in de webbrowser in google chrome I don't have this problem. is this normal?

Yes the problem seems to be with web versions not the application itself on desktop.

**Comment by itsmhashim** [comment]
> Experienced the same problem, Brave is blocking something needed for this, click on Lionface in the toolbar and turn off shields for app.stremio.com.

I have tried that as well but no use.

**Comment by TRtomasz** [comment]
> > what do you see when you open http://127.0.0.1:11470/ if your browser?
> 
> I don't see anything the page doesn't open.

then something on your desktop is blocking the streaming service and it is not running
proxy, firewall, antivirus

**Comment by daaf85** [comment]
i have solved my issue in the web version. I have followed the instructions in step 5 in this guide: https://howtomediacenter.com/en/stremio-guide/#5-stremio-web-and-the-streaming-server.

If I haven't used the Stremio desktop version in my MacBook Pro i have to install the app 'Stremio Service' to solve this problem. See this blog: https://blog.stremio.com/using-stremio-service/.

**Comment by itsmhashim** [comment]
> i have solved my issue in the web version. I have followed the instructions in step 5 in this guide: https://howtomediacenter.com/en/stremio-guide/#5-stremio-web-and-the-streaming-server.
> 
> If I haven't used the Stremio desktop version in my MacBook Pro i have to install the app 'Stremio Service' to solve this problem. See this blog: https://blog.stremio.com/using-stremio-service/.

Hey thank you so much this seems to be work however the streams are now too slow take hours to load. I checked TPB streams.

**Comment by Pwnz74** [comment]
that think works do it via Web version if you got Brave browser. Tutorial on the top - step 5

- Copy past to browser:  http://127.0.0.1:11470/
- Log into your account
- On Stremio Web, go to Settings, select Streaming. Press the Enable remote https server dropdown and select an IP address and copy it
- Paste it into desktop Streamio
- Restart desktop Streamio and it should worked with your customer IP

I think it is connected that Brave forcing HTTPS protocol and it messing Streamio
